### DISCUSSION TITLE
Compatibilité Linux ou web

### DISCUSSION THREAD

**Discussion by Celestial04** [Lenochxd/WebDeck#23] [Ideas]
Ça serait bien d'ajouter une compatibilité pour les distribs Linux, ou de faire en sorte que ça soit un serveur Node qui gère tout

**Comment by Lenochxd** [comment]
Hey :)
La compatibilité Linux est en cours, je suis en train de tout restructurer pour que ça soit plus simple à implémenter, je compte faire la compatibilité macos en même temps. Bonne idée pour le mode web mais ça serait grave complexe niveau sécurité alors il faudra voir, c'est pas pour bientôt en tout cas

**Comment by Lenochxd** [comment]
#35
